src/docfx/build/xref/XrefResolver.cs

```diff
@@ -324,7 +324,10 @@ private static string RemoveSharingHost(string url, string hostName)
                 var dependencyType = GetDependencyType(referencingFile, spec);
                 _dependencyMapBuilder.AddDependencyItem(referencingFile, spec.DeclaringFile, dependencyType);
 
-                var href = UrlUtility.GetRelativeUrl(_documentProvider.GetSiteUrl(inclusionRoot), spec.Href);
+                // Output absolute URL starting from Architecture and TSType
+                var mime = _documentProvider.GetMime(inclusionRoot);
```

Comment:
consider extract this like `TemplateEngine.IsConceptual`, and also use `Equals() || Equals()` removes this `string[]` allocation.